sources/packages/backend/apps/api/src/app.aest.module.ts

```diff
@@ -63,6 +63,7 @@ import {
   ConfirmationOfEnrollmentAESTController,
   ConfirmationOfEnrollmentControllerService,
   OverawardAESTController,
+  OverAwardControllerService,
```

Comment:
As we use overaward as one word in application, I wish it should follow the same pattern for OverAwardControllerService (OverawardControllerService) similar to OverawardAESTController.